src/dataSources/cloudFirestore/partner.js

```diff
@@ -24,7 +24,7 @@ function partnerCollection(dbInstance) {
       ...d.data(),
     }));
 
-    const sortBy = ['PIONEER', 'EXPLORER', 'SCOUT', 'MEDIA'];
+    const sortBy = ['PIONEER', 'EXPLORER', 'SCOUT', 'CUB', 'PATRION', 'MEDIA'];
```

Comment:
shouldn't it be `PATRON`?

Reply:
LOL

Reply:
actually we should also add last years levels

Reply:
just add these to the end of that array and it would work out.  PLATINUM, GOLD, SILVER, BRONZE

Reply:
Resolved last push

Reply:
@csell5 at end, or before MEDIA?

Reply:
Previous levels added